Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -4262,17 +4262,29 @@ - (UIColor *)insertionPointColor
 #if PLATFORM(MACCATALYST)
     return [self _cascadeInteractionTintColor];
 #else
+#if HAVE(UI_ASYNC_TEXT_INTERACTION)
+    if (self.shouldUseAsyncInteractions)
+        return self.extendedTraitsDelegate.insertionPointColor;
+#endif // HAVE(UI_ASYNC_TEXT_INTERACTION)
```

Comment:
Optional nit for uniformity, either just `#endif` here or `#endif // HAVE(UI_ASYNC_TEXT_INTERACTION)` in the next three changes?

Reply:
Ah, I put a `// HAVE(UI_ASYNC_TEXT_INTERACTION)` here just for clarity, since it's already nested inside another `#if - #elif`, but thought the other `#if`s below read clearly enough. I'll remove the `// HAVE(UI_ASYNC_TEXT_INTERACTION)` here for consistency.